### DISCUSSION TITLE
[Translate FR] For the future release ? When ?

### DISCUSSION THREAD

**Discussion by jujudusud** [jamulussoftware/jamulus#1527] [Documentation]
Hi here,

I don't know where to search for information that will allow me to know when we can start translating the website into French and also the date for the next release.
Is that someone who can help me?

**Comment by ignotus666** [comment]
We usually get notified when docs and UI text are frozen, and have a couple of weeks or more to do the translations, don't worry.

**Reply by jujudusud** [reply]
Thanks @ignotus666 :-)
We have also the screenshots to do for the next release !
ok perfect.
